trueos-handbook/install.rst

```diff
@@ -9,6 +9,290 @@ graphical desktop directly onto a hard drive or into a virtual machine
 using virtualization software such as
 `VirtualBox <https://www.virtualbox.org/>`_.
 
+.. index:: Download and Prepare to Install
+.. _Download and Prepare to Install:
+
+Download and Prepare to Install
+===============================
+
+|trueos| uses a rolling release model rather than versioned releases.
+There are two primary options of |trueos| install: STABLE and UNSTABLE:
+
+1. STABLE is meant to be less frequently updated and synchronized with
+   FreeBSD. This means users see less experimental work and generally
+   have a smoother experience. However, users on STABLE also typically
+   wait longer for bugfixes and patches to be available. While some
+   |trueos| development may be backported to STABLE early, FreeBSD
+   patches and port synchronization is done on a six-month schedule.
+
+2. UNSTABLE is the full leading edge of TrueOS and FreeBSD development.
+   Patches are very frequent, but can incorporate experimental work from
+   both the |trueos| and FreeBSD projects. UNSTABLE is recommended for
```

Comment:
both the |trueos| and FreeBSD projects. -> |trueos|, FreeBSD, and other Open Source projects and contributors.